clusters/app.ci/publicize/_config.yaml

```diff
@@ -0,0 +1,181 @@
+repositories:
+  openshift-priv/ansible-service-broker: openshift/ansible-service-broker
```

Comment:
@droslean why is this list btw needed, we can just derive the one from the other or not?

Reply:
This is a configuration file for the publicize plugin. see https://github.com/openshift/ci-tools/pull/872

Reply:
Yeah but why does the publicize plugin not simply infer the public name from the private name by stripping `-priv` from the org? Then we save ourselves the configfile, the tool to update the configfile and the periodic.

Reply:
Because the org can be different than `openshift`.